includes/Plugin.php

```diff
@@ -158,6 +158,8 @@ function() use ( $options, $transients, $assets ) {
 				( new Core\Notifications\Notifications( $this->context, $options, $authentication ) )->register();
 				( new Core\Util\Debug_Data( $this->context, $options, $user_options, $authentication, $modules ) )->register();
 				( new Core\Admin\Standalone( $this->context ) )->register();
+				( new Core\Util\Migration_1_3_0( $this->context, $options, $user_options ) )->register();
+				( new Core\Util\Migration_1_8_1( $this->context, $options, $user_options, $authentication ) )->register();
```

Comment:
Noting the 1_3_0 migration class is moved inside the `init` callback here. I assume that is fine, maybe better, and I assume these need to load in order.

Reply:
Yeah they need to load in order, that's why I moved this too. Furthermore, we were previously not passing the `User_Options` instance to `Migration_1_3_0` (because the outer method does not have awareness of it), which was technically okay, but still something we shouldn't do, to make sure we consistently use the same instances in the plugin.